src/PlusDataCollection/CMakeLists.txt

```diff
@@ -1824,6 +1825,61 @@ IF(PLUS_USE_OpenCV_VIDEO)
     )
 ENDIF()
 
+#---------------------------------------------------------------------------
+# InfraredSeekCam
+IF(PLUS_USE_INFRARED_SEEK_CAM)
+  # Check OpenCV 
+  IF(NOT PLUS_USE_OpenCV)
+    MESSAGE("PLUS_USE_INFRARED_SEEK_CAM requires PLUS_USE_OpenCV option enabled, enabling.")
```

Comment:
This may need to be done at an earlier stage, or a FIND_PACKAGE(OpenCV REQUIRED) must also be included in this block. Otherwise CMake might continue without actually having OpenCV.

Reply:
We copied this part of code from OpenCV_Video. Could you specify another way?